recipe/meta.yaml

```diff
@@ -23,28 +24,29 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - python
-    - pip <22.1
+    - pip
     - numpy {{ numpy }}
     - cython
     - packaging
 
   run:
     - python
-    - scipy >=1.0.0
-    - {{ pin_compatible('numpy', lower_bound='1.18') }}
+    - scipy >=1.5.0
+    - {{ pin_compatible('numpy', lower_bound='1.19') }}
     - matplotlib-base >=1.5.1
     - seaborn
     - netcdf4 >=1.0
```

Comment:
Going by https://github.com/MDAnalysis/mdanalysis/pull/3025/files I'm going to say `2.10.0` (since we used to pin to that version specifically). We should officially increase the minimum version in the core repo though (3.0.0 would be ~ NEP29 style, i.e. last 2 years)..

Reply:
I am good with >=2.1 for right now. Thanks for starting the discussion in our issue tracker.